src/Serializer/AbstractItemNormalizer.php

```diff
@@ -638,6 +639,13 @@ protected function getAttributeValue(object $object, string $attribute, string $
                 $childContext = $this->createChildContext($context, $attribute, $format);
                 unset($childContext['iri'], $childContext['uri_variables'], $childContext['resource_class'], $childContext['operation']);
 
+                if (null !== $itemUriTemplate = $propertyMetadata->getUriTemplate()) {
+                    $operation = $this->resourceMetadataCollectionFactory->create($resourceClass)->getOperation($itemUriTemplate, true, true);
```

Comment:
```suggestion
                    $operation = $this->resourceMetadataCollectionFactory->create($resourceClass)->getOperation($itemUriTemplate, true, true);
```

you can use named arguments for clarity on these booleans?